torch/_inductor/codecache.py

```diff
@@ -2322,8 +2373,19 @@ def load_async(cls, source_code: str, cuda=False, submit_fn=None, extra_flags=()
             "vec_isa": pick_vec_isa(),
             "extra_flags": extra_flags,
         }
-        cpp_command = repr(cpp_compile_command("i", "o", **compile_command))
-        key, input_path = write(source_code, "cpp", extra=cpp_command)
+
```

Comment:
Thanks to @int3 , adding `_set_gpu_runtime_env()  # cpp_extension consults the env` here makes fbcode tests pass.